CHANGELOG.md

```diff
@@ -5,6 +5,24 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [0.11.0] - 2022-02-10
+
+### Added
+- Add MySQLCluster v1beta2 (#350, #359)
```

Comment:
I think this description is insufficient.
#350 is an important new feature. Add primaryServiceTemplate and replicaServiceTemplate.
#359 is just a bug fix.

How about explaining the migration for the upgrade of MySQLCluster?
(You only need to change the version of your MySQLCluster, thanks to the conversion webhook.)

Reply:
Thank you. I wrote about MySQLCluster v1beta2.

I don't think #359 (and #376) need not list individually. They are part of the implementation of MySQLCluster v1beta2.